refinery/core/test_api.py

```diff
@@ -407,14 +407,6 @@ def test_get_node_list_count(self):
         self.assertEqual(node_list_dict['meta']['total_count'], 1)
         self.assertEqual(len(node_list_dict['objects']), 1)
 
-    def test_get_node_detail_fields(self):
-        response = self.api_client.get(api_uri(NodeResource, self.node.uuid))
-        node_fields = self.deserialize(response).keys()
-        expected_fields = NodeResource._meta.fields + [
-            'file_import_status', 'resource_uri'
-        ]
-        self.assertItemsEqual(node_fields, expected_fields)
-
```

Comment:
Should this be removed or just modified?

Reply:
I'd say just remove it. In this milestone, all these tests will be deleted and new tests will be written for any new DRF APIs or have been written. We already have a Node API V2 which will need to be extended for more detail fields. There shouldn't be any roadblocks removing the node tasty pie API.